src/Nethermind/Nethermind.Consensus.AuRa/Transactions/TxPermissionFilter.cs

```diff
@@ -91,7 +92,7 @@ private ITransactionPermissionContract.TxPermissions GetPermissions(Transaction
                 {
                     (txPermissions, shouldCache) = versionedContract.AllowedTxTypes(parentHeader, tx);
                 }
-                catch (AuRaException e)
+                catch (AbiException e)
```

Comment:
you change AbiException to AuRa and  you changed the exception that you catch from AuRa to Abi. Was it intended?